src/flag_gems/fused/gelu_and_mul.py

```diff
@@ -19,6 +19,28 @@ def gelu_none_and_mul_kernel(x, y):
     return x_gelu * y
 
 
+@pointwise_dynamic(
+    promotion_methods=[(0, 1, 2, "DEFAULT"), (0, 1, 2, "DEFAULT")], num_outputs=2
+)
+@triton.jit
+def gelu_none_and_mul_grad_kernel(x, y, dgrad):
+    x_fp32 = x.to(tl.float32)
+    x_gelu = 0.5 * x_fp32 * (1 + erf(x_fp32 * 0.7071067811))
```

Comment:
Shall we give a named symbol to the constant? Or just 2**-0.5

Reply:
Name this constant as `RCP_SQRT_2`